consensus/phases/states.go

```diff
@@ -56,9 +56,12 @@ import (
 	"github.com/insolar/insolar/insolar"
 	"github.com/insolar/insolar/network"
 	"github.com/insolar/insolar/network/merkle"
+	"github.com/insolar/insolar/network/node"
 )
 
 type FirstPhaseState struct {
+	*ConsensusState
```

Comment:
Are you sure `ConsensusState ` should be here and not just passed into phases from outside?
I think we could initialize it in `PhaseManager` and then just pass it to phases.

Reply:
We can't initialize `ConsensusState` in `PhaseManager` when a node is joiner (then `ConsensusState` is fully initialized during phase 1)

Reply:
There's only one usage of `NewConsensusState`. Could you please explain it more clearly?

Reply:
`NewConsensusState` returns not fully initialized state and it is fully initialized during phase 1. I don't want to expose partly initialized objects to the phase manager abstraction level